constants/constants.go

```diff
@@ -4,5 +4,7 @@ const (
 	CTX_DB            = "db"
 	CTX_DOCKER_CLIENT = "docker_client"
 
-	BUILD_TIMEOUT = 5
+	BUILD_TIMEOUT   = 5
+	BUILD_CPU_SHARE = 102      // cpu share relative to the default (1024) for build containers
```

Comment:
Nah, these are the build containers and theyll compete with our server for resources. Ideally we run these build containers on another cluster, but I figure limiting them to 10% of server resources should be okay for now. We can always tune later.